code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -93,7 +93,7 @@
 					target.visible_message("<span class='danger'>[user] is trying to take a blood sample from [target]!</span>", \
 									"<span class='userdanger'>[user] is trying to take a blood sample from [target]!</span>")
 					busy = TRUE
-					if(!do_mob(user, target, extra_checks=CALLBACK(L, /mob/living/proc/can_inject,user,1)))
+					if(!do_mob(user, target, extra_checks=CALLBACK(L, /mob/living/proc/can_inject,user,1, penetrate_thick = proj_piercing)))
```

Comment:
travis doesn't seem to like this line for some reason? I think it doesn't enjoy the keyword usage here because it thinks its supposed to be the keyword for CALLBACK